java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -747,7 +751,7 @@ void testFromScalarZeroRows() {
         case TIMESTAMP_MILLISECONDS:
         case TIMESTAMP_MICROSECONDS:
         case TIMESTAMP_NANOSECONDS:
-          s = Scalar.timestampFromLong(type, 1234567890123456789L);
+          s = Scalar.timestampFromLong(new DType(type, 0), 1234567890123456789L);
```

Comment:
This is why we want a factory because now the test is creating a new DType class when it does not need to.

Reply:
done